### PR TITLE
checksum validation implemented

### LINKED ISSUE #32: implement validateMd5()
this can be not blocked if we have a (random) fixture file that is valid to md5 checksum;  we can also check invalid behavior by changing the checksum.

### LINKED ISSUE #33: implement validateSha1()
this can be not blocked if we have a (random) fixture file that is valid to sha1 checksum;  we can also check invalid behavior by changing the checksum.

### PR DESCRIPTION
**Pull request by ndushay**
closes #32 
closes #33 

Since it turned out these were static methods, it seemed best to add them to WasapiDownloader ... no where else seemed particularly better, and there's an outside chance we would add a way to use command line arguments to allow validation of a file from the command line.

### CODE REVIEW

File: test/edu/stanford/dlss/was/TestWasapiDownloader.java [hunk 1]
```diff
@@ -9,18 +11,45 @@
 
 public class TestWasapiDownloader {
 
-    @Before
```
Comment (tingulfsen) [inline test/edu/stanford/dlss/was/TestWasapiDownloader.java]: I like the 4 indents that were already present in this file - is there a particular reason to change it?

Comment (tingulfsen) [reply test/edu/stanford/dlss/was/TestWasapiDownloader.java]: Never mind, I see now that it's like that elsewhere too.

Comment (ndushay) [reply test/edu/stanford/dlss/was/TestWasapiDownloader.java]: nearly all the other java files are 2 spaces indent.  I was just making it consistent.  @jmartin-sul ... what say you?

**Comment by ndushay** [comment]
@tingulfsen I think it's fine for you to merge this if you're ok with it.

**Review by jmartin-sul — APPROVED** [review]

**Pull request merged**